packages/legend-studio/src/models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_DomainSerializationHelper.ts

```diff
@@ -352,6 +352,12 @@ export const V1_associationSchema = createModelSchema(V1_Association, {
   name: primitive(),
   package: primitive(),
   properties: list(usingModelSchema(V1_propertySchema)),
+  // NOTE: we don't process milestoning at the moment so this is added to ensure
```

Comment:
Probably we don't need this comment here, we already have the marker

Reply:
this comment was added to make it consistent with [this snippet]( https://github.com/kshradhan/legend-studio/blob/1686faef2a94cce45c901dd1a380d42f3a71af85/packages/legend-studio/src/models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_DomainSerializationHelper.ts#L278)

Reply:
ah we can delete them altogether now